cli/src/commands/clean.rs

```diff
@@ -30,35 +30,38 @@ pub struct Clean {
     /// Enables development mode, specify the unique ID of the local node to clean.
     #[clap(long)]
     pub dev: Option<u16>,
-    /// Specify the path to a directory containing the ledger
-    #[clap(long = "path")]
-    pub path: Option<PathBuf>,
+    /// Specify the path to a directory containing the storage database for the ledger.
```

Comment:
```suggestion
    /// Specify the path to a directory containing the storage for the ledger.
```
nit: "storage database" is a bit redundant IMO

Reply:
(also reverted)